accounting/localizations/argentina.rst

```diff
@@ -0,0 +1,280 @@
+========
+Argentina
+========
+
+Introduction
+~~~~~~~~~~~~
+The Argentinean localization has been improved and extended in Odoo V13, in this version the next modules are available:
+
+#. **l10n_ar**:  This module add accounting features for the Argentinian localization, which represent the minimal configuration needed for a company  to operate in Argentina and under the AFIP (Administración Federal de Ingresos Públicos) regulations and guidelines.
```

Comment:
please keep 80 char for long lines, it makes it easier to read the diff if you change one letter afterwards